forge-gui/res/cardsfolder/d/drop_tower.txt

```diff
@@ -0,0 +1,15 @@
+Name:Drop Tower
+ManaCost:no cost
+Types:Artifact Attraction
+Variant:A:Lights:2 3 6
+Variant:B:Lights:2 4 6
+Variant:C:Lights:2 5 6
+Variant:D:Lights:3 4 6
+Variant:E:Lights:3 5 6
+Variant:F:Lights:4 5 6
+K:Visit:TrigPump
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature | KW$ Flying | SubAbility$ DBAnimate | SpellDescription$ Target creature gains flying until end of turn, or until any player rolls a 1, whichever comes first.
+SVar:DBAnimate:DB$ Animate | Defined$ Targeted | Triggers$ DiceRolled
+SVar:DiceRolled:Mode$ RolledDie | Execute$ TrigRemove | ConditionPresent$ Card.Self+withFlying | ValidPlayer$ You | ValidResult$ EQ1 | Static$ True
```

Comment:
bad condition (actually this solution is rules breaking anyway, but Unfinity...)